integration/nwo/deploy.go

```diff
@@ -242,7 +241,11 @@ func ApproveChaincodeForMyOrg(n *Network, channel string, orderer *Orderer, chai
 				InitRequired:        chaincode.InitRequired,
 				CollectionsConfig:   chaincode.CollectionsConfig,
 				ClientAuth:          n.ClientAuthRequired,
-			})
+			}
+			if p.DevMode {
+				cmd.PackageID = ""
+			}
```

Comment:
We don't need this after your update, right? And then we can revert the session and command code to its original state.

Reply:
yes, correct.